terraform/appveyor-s3-cache/main.tf

```diff
@@ -0,0 +1,31 @@
+provider "aws" {
+  region = "us-west-1"
+}
+
+terraform {
+  backend "s3" {
+    bucket  = "iohk-ops-tfstate"
```

Comment:
@domenkozar, is this snippet with the TF state S3 bucket intended to be copied to all TF deployments, or maybe we could separate it into a separate module, outside of the `appveyor-s3-cache`-specific part?

Reply:
Yes, that's going to be shared between deployments. We can do that later on, once we expand tf usage

Reply:
@domenkozar @deepfire It is common to use the same bucket for all deployments done with TF, although having a __separate__ TF state file is a good practice. So, I'd recommend for each logical deployment to have it's own snippet like that, granted with the same bucket name. Reasoning for this is fairly straightforward, you don't want to have one global state for your whole infrastructure. Of course, this is just a suggestion. :)

Reply:
@lehins thanks!